frontends/infinite-corridor/src/components/FieldMenu.test.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react"
+import { BrowserRouter } from "react-router-dom"
+import { render, screen } from "@testing-library/react"
+
+import { urls } from "../api/fields"
+import * as factories from "../api/fields/factories"
+import { setMockResponse, user } from "../test-utils"
+import FieldMenu from "./FieldMenu"
+
+describe("FieldMenu", () => {
+  it("Includes a link to the FieldChannel edit page", async () => {
+    const field = factories.makeField()
+    setMockResponse.get(urls.fieldDetails(field.name), field)
+    render(
+      <BrowserRouter>
+        <FieldMenu field={field} />
+      </BrowserRouter>
```

Comment:
I would suggest using MemoryRouter in jest tests, not BrowserRouter because (emphasis mine):

> You can also use BrowserRouter if your test environment has the browser globals window.location and window.history (which is the default in Jest through JSDOM, **but you cannot reset the history between tests**).

I would recommend:
```tsx
const history = createMemoryHistory()
render(
  <Router history={history}>
    <FieldMenu field={field} />
  </Router>
)
```

Reply:
Thanks for the tip!